core/arch/arm/plat-zynq7k/plat_init.S

```diff
@@ -102,8 +102,8 @@ UNWIND(	.fnstart)
 	movt r0, #0x0002
 	write_nsacr r0
 
-	movw r0, #0x0000
-	movt r0, #0x0001
+	movw r0, #0x0001
```

Comment:
@jenswi-linaro, by the way I started to switch other occurrences of `movt/movw` into `mov_imm` (plat-imx, plat-ls, plat-stm, plat-zynq7k) although there was no issue reported on these. Do you think it's a good idea ? i think it is, but i sometime tend to out-scope my prerogatives :)